flocker/node/agents/functional/test_gce.py

```diff
@@ -0,0 +1,97 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Functional tests for ``flocker.node.agents.gce`` using a GCE cluster.
+
+In order to run these tests you'll need to define the following environment
+variables:
+
+``FLOCKER_FUNCTIONAL_TEST=TRUE
+FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE=$HOME/acceptance.yml
+FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER=gce``
+
+The configuration stanza for the GCE backend should resemble:
+``
+gce:
+    zone: <gce-region>
+    project: <gce-project-name>
+``
+
+Note that, at this time, authentication is done using the implicit VM service
+account.  When creating your GCE instance be sure to check ``Allow API access
+to all Google Cloud services in the same project.``
+
+"""
+
+from uuid import uuid4
+
+from ..test.test_blockdevice import (
+    make_iblockdeviceapi_tests
+)
+
+from ..test.blockdevicefactory import (
+    ProviderType, get_blockdeviceapi_with_cleanup,
+    get_minimum_allocatable_size, get_device_allocation_unit
+)
+
+from ....testtools import TestCase
```

Comment:
Clarified offline. No action required, jml is lamenting the use of relative imports. I have sympathy for the feeling that relative imports are hard to maintain, but they seem to be standard in the flocker codebase so probably going to leave as is.